app/scripts/models/resume-token.js

```diff
@@ -19,7 +19,9 @@ define([
       campaign: undefined,
       entrypoint: undefined,
       state: undefined,
-      verificationRedirect: undefined
+      verificationRedirect: undefined,
+      // fields from a User
+      uniqueUserId: undefined
```

Comment:
I ignored my own convention so that a casual reader could easily delineate which fields came from a Relier and which from a User.